website/content/docs/secrets/databases/mongodbatlas.mdx

```diff
@@ -17,6 +17,8 @@ it uses a public and private key pair to authenticate.
 See the [database secrets engine](/vault/docs/secrets/databases) docs for
 more information about setting up the database secrets engine.
 
+Please do not confuse this with MongoDB Atlas secrets engine described [here](/vault/docs/secrets/mongodbatlas).
```

Comment:
<Note>
  The information below relates to the MongoDB Altas <b>database plugin</b> for the Vault database secrets engine.
  Refer to the <a href="vault/docs/secrets/mongodbatlas">MongoDB Atlas secrets engine</a>
  for information about using the MongoDB Atlas secrets engine for the Vault.
</Note>